### PR TITLE
CODENVY-741 rename group id for site

### PR DESCRIPTION
**Pull request by vkuznyetsov**
Rename group ID for site
https://github.com/codenvy/codenvy/issues/741

### CODE REVIEW

**Comment by vkuznyetsov** [comment]
ci-build

File: wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/pom.xml [hunk 1]
```diff
@@ -193,8 +192,7 @@
                                         che-core-api-user,
                                         che-core-api-machine,
                                         che-core-api-workspace,
-                                        che-core-api-ssh
-                                    </includeArtifactIds>
```
Comment (garagatyi) [inline wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/pom.xml]: Please fix formatting

Comment (vkuznyetsov) [inline wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/pom.xml]: I run mvn sortpom:sort command. It formatted a pom file on its own. Do you think we need to format code each time we run the sortpom?

Comment (garagatyi) [inline wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/pom.xml]: Then its ok

**Pull request merged**